### ISSUE TITLE
Compiled output is owned by root

### ISSUE THREAD

**Issue by madonius** [kapicorp/kapitan-reference#27]
When checking out the repository and running the compile command the output directory and all it's contents are owned by root.
```
$ > uname -a
Linux aretousa 5.6.0-2-amd64 #1 SMP Debian 5.6.14-1 (2020-05-23) x86_64 GNU/Linux
$ > sudo rm -rf compiled
$ > ./kapitan compile
Compiled kapicorp/tesoro (1.21s)
Compiled examples/tutorial (1.50s)
Compiled examples/pritunl (1.39s)
Compiled examples/echo-server (1.20s)
Compiled kapicorp/dev-sockshop (3.53s)
Compiled kapicorp/prod-sockshop (3.61s)
Compiled examples/global (0.56s)
Compiled examples/examples (1.85s)
Compiled examples/mysql (1.28s)
Compiled examples/gke-pvm-killer (0.98s)
Compiled examples/postgres-proxy (0.98s)
Compiled examples/sock-shop (3.21s)
$ > ls -alh
…
drwx------ 14 root        root        4.0K Jun 17 11:30 compiled
…
```

**Comment by ramaro** [comment]
Good find, sounds like we're missing the `-u $UID` docker option there. (As documented in https://kapitan.dev/#quickstart) @ademariag

**Comment by madonius** [comment]
Fixed by #28
